.changes/next-release/bugfix-AmazonDynamoDBEnhacned-c4253dc.json

```diff
@@ -0,0 +1,5 @@
+{
+    "category": "Amazon DynamoDB Enhacned", 
+    "type": "bugfix", 
+    "description": "innerToString method inside EnhancedType raises a NPE when built using documentOf as `rawClassParameters` is set to `null`."
```

Comment:
Can we rephrase this to what the new behavior is (no longer throws an NPE...), as opposed to what the problem was?